stable/grafana/values.yaml

```diff
@@ -146,9 +146,30 @@ grafana.ini:
     data: /var/lib/grafana/data
     logs: /var/log/grafana
     plugins: /var/lib/grafana/plugins
+    provisioning: /etc/grafana/provisioning
   analytics:
     check_for_updates: true
   log:
     mode: console
   grafana_net:
     url: https://grafana.net
+
+## Sidecars that collect the configmaps with specified label and stores the included files them into the respective folders
+## Requires at least Grafana 5 to work and can't be used together with parameters dashboardProviders, datasources and dashboards
+sidecar:
+  image: kiwigrid/k8s-sidecar:0.0.3
```

Comment:
Suggest sidecar image should be provided on values.yaml as default. Please don't hardcode it.